packages/camera/camera_avfoundation/ios/camera_avfoundation/Sources/camera_avfoundation/FLTCam.m

```diff
@@ -88,10 +85,10 @@ @interface FLTCam () <AVCaptureVideoDataOutputSampleBufferDelegate,
 @property(assign, nonatomic) int maxStreamingPendingFramesCount;
 
 @property(assign, nonatomic) UIDeviceOrientation lockedCaptureOrientation;
-@property(assign, nonatomic) CMTime lastVideoSampleTime;
-@property(assign, nonatomic) CMTime lastAudioSampleTime;
-@property(assign, nonatomic) CMTime videoTimeOffset;
-@property(assign, nonatomic) CMTime audioTimeOffset;
+@property(assign, nonatomic) CMTime recordingTimeOffset;
+@property(assign, nonatomic) CMTime lastSampleEndTime;
+@property(nonatomic) AVCaptureOutput *outputForOffsetAdjusting;
+@property(assign, nonatomic) CMTime lastAppendedVideoSampleTime;
```

Comment:
is `lastAppendedVideoSampleTime` the same as `lastSampleEndTime`? 

Can you add some doc on these 4 properties?